c/cert/test/rules/EXP43-C/test.c

```diff
@@ -0,0 +1,91 @@
+#include <stddef.h>
```

Comment:
minor suggestion: do we want to add this compliant testcase:
```
int *restrict g11;
int * g21;

void test_global_local1() {
  g11 = g21;               // COMPLIANT
}
```

Reply:
Added.